content-repo/extra-docs/articles/microsoft-integrations-overview.md

```diff
@@ -0,0 +1,189 @@
+Manage groups in Azure Active Directory and O365. 
+- List, create, and update groups. 
+- List, add, and remove members. 
+
+### [Azure Active Directory Identity And Access](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-identityand-access) 
+Manage Active Directory roles and role members.
+
+### [Azure Active Directory Applications / Service Principals](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-applications) 
+Manage applications and service principals. 
+
+## O365
+
+### Use Cases
+
+* Download a file from OneDrive.
+* Send a message via Microsoft Teams.
+* Add a member to an existing team.
+* Schedule an event in the calendar.
+
+### [O365 File Management (Onedrive/Sharepoint/Teams)](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-files)
+Manage files in O365 (OneDrive/SharePoint/Teams).
+- Upload and download files.
+- List drive and folder content.
+- List SharePoint sites.  
+
+### [Microsoft Teams](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams) 
+Enable communicating and mirroring via Microsoft Teams.
+- Create and update channels.
+- Add users to channel.
+- Message users.
+- Ring user.
+- Message mirroring.
+
+### [Microsoft Teams Management](https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams-management) 
+Manage teams and team members
+- Create and update teams.
+- Add and remove team members.
+
+### [O365 Outlook Calendar](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-calendar) 
+Manage calendar events.
+
+### [Microsoft Graph API](https://xsoar.pan.dev/docs/reference/integrations/microsoft-graph-api) 
+This is a generic integration that supports running any endpoint of MS Graph API.
+Since the API is very vast and not all of the endpoints are implemented, this integration can be used.
+
+
+## Audit Logs 
+
+### Use Cases:
+* Find failed login events
+* Find publicly shared files events
+* Find security events
+
+### [Microsoft Management Activity](https://xsoar.pan.dev/docs/reference/integrations/microsoft-management-activity-api-o365-azure-events) 
+Ingest events from O365 (Azure AD, SharePoint, EWS, etc) as incidents.
+
+### [Microsoft Policy And Compliance - Office365 And Azure Audit Log](https://xsoar.pan.dev/docs/reference/integrations/microsoft-policy-and-compliance-audit-log)
+Search audit logs from O365 and Azure.
+
+-----
+
+## Exchange and EWS 
+
+### Use Cases
+* Find an email message.
+* Move an email message to a different folder.
+* Send an email.
+* Modify your Outlook recipient list.
+
+| Integration 	| Cloud/On-Prem 	| Usage 	| Limitations 	| Auth Method 	|
+|:---:	|:---:	|:---:	|:---:	|:---:	|
+| [EWS Mail Sender](https://xsoar.pan.dev/docs/reference/integrations/ews-mail-sender)	| On-Prem 	| Notify user by mail in a playbook.<br>Fetch emails as incidents. 	| Supports basic authentication only 	| Basic Auth 	|
```

Comment:
I'm not sure you can use the <br> without closing it
```
ValueError: Failed verfiying via MDX server. Status: 500. Error: MDX parse failure: SyntaxError: unknown: Expected corresponding JSX closing tag for <br> (108:116)

  106 | <td parentName="tr" {...{"align":"center"}}><a parentName="td" {...{"href":"https://xsoar.pan.dev/docs/reference/integrations/ews-mail-sender"}}>{`EWS Mail Sender`}</a></td>
  107 | <td parentName="tr" {...{"align":"center"}}>{`On-Prem`}</td>
> 108 | <td parentName="tr" {...{"align":"center"}}>{`Notify user by mail in a playbook.`}<br>{`Fetch emails as incidents.`}</td>
```

Reply:
```suggestion
| [EWS Mail Sender](https://xsoar.pan.dev/docs/reference/integrations/ews-mail-sender)	| On-Prem 	| Notify user by mail in a playbook.</br>Fetch emails as incidents. 	| Supports basic authentication only 	| Basic Auth 	|
```